simplify/src/main/java/org/cf/simplify/SimplifyOptionsParser.java

```diff
@@ -108,6 +110,11 @@ private static SimplifyOptions getSimplifyOptions(CommandLine cmd) throws ParseE
             simplifyOpts.setOutFile(new File(outFileName));
         }
 
+        if (Main.errorDexFile != null) {
```

Comment:
This is a bit odd. What about making the error dex file a temporary file and assuming it gets deleted after simplify stops?